site/docs/drivers/kube-docker.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 1
+title: Kubernetes Docker-in-Docker Driver
+---
+
+# Kubernetes Docker-in-Docker driver
```

Comment:
I think in general, kube-docker might be the wrong name here, as this plugin can use different strategy in the future.
In addition, Docker-In-Docker is not really used. It's just Docker. (affectively, the pod is also not running in Docker)
Like Tekton that uses Docker for building: https://github.com/tektoncd/catalog/tree/main/task/docker-build/0.1

Reply:
what different strategy do you have in mind?
I wanted to differentiate this driver from another we might have in the future which will use native kubernetes resources.